hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/SimpleContainerDownloader.java

```diff
@@ -135,4 +152,21 @@ protected CompletableFuture<Path> downloadContainer(
   public void close() {
     // noop
   }
+
+  public Path getWorkingDirectory() {
+    Path defaultWorkingDirectory =
+        Paths.get(System.getProperty("java.io.tmpdir"));
```

Comment:
1. Since data volume must be configured, then we don't fallback to this "java.io.tmpdir". 
2.  move this volume choose logic to outside DownloadAndImportReplicator, so we can use this choose volume later the container import process.